lib/PuppeteerSharp/Constants.cs

```diff
@@ -0,0 +1,160 @@
+﻿namespace PuppeteerSharp
+{
+    /// <summary>
+    /// Constants common to devtools API interaction
+    /// </summary>
+    internal class Constants
```

Comment:
not sure how I feel about this class since it's not ordered, I would have liked it to be ordered if possible but I'm not sure it's possible since many of the constants are used in multiple locations throughout the project

Reply:
I'm not sure I understand the need for it to be ordered ?